M2/Macaulay2/editors/prism/macaulay2.ts.in

```diff
@@ -0,0 +1,32 @@
+import Prism from "prismjs";
```

Comment:
Should we include this line?  It wasn't before, and it doesn't appear that other prism language definition files include it: https://github.com/PrismJS/prism/tree/master/components